code/game/objects/items/devices/radio/radio.dm

```diff
@@ -585,4 +592,23 @@
 	. = ..()
 	set_listening(FALSE)
 
+/obj/item/radio/ui_action_click(mob/user, action)
+	if(emped)
+		to_chat(user, span_warning("The panic button is broken!"))
+		return
+
+	panic(user)
+
+/// The panic button, alerts Security channel that the user currently needs help
+/obj/item/radio/proc/panic(mob/user)
+
+	if(panic_time > world.time)
+		to_chat(user, span_warning("The panic button cooldown didn't pass yet!"))
```

Comment:
```suggestion
		balloon_alert(user, "on cooldown!")
```
Better to go with balloon alerts on visible things like this, especially with the context that it'd be more combat focused, so they keep their attention to the middle of the screen.